x-pack/solutions/search/packages/kbn-search-index-documents/components/result/result.tsx

```diff
@@ -65,68 +66,79 @@ export const Result: React.FC<ResultProps> = ({
         });
   const toolTipContent = <>{tooltipText}</>;
 
+  const showResultsFields = isExpanded
+    ? fields.length > 0
+    : fields.slice(0, defaultVisibleFields).length > 0;
```

Comment:
do we need to do the slice here? I feel like `defaultVisibleFields > 0 && fields.length > 0` would give us the same result without having to do any array operations.

Reply:
updated in https://github.com/elastic/kibana/pull/217243/commits/c5e377796e25a43b6f9a50e2428c4b645ee3029a